docker-compose-with-backend.yml

```diff
@@ -0,0 +1,16 @@
+name: chatbot
+
+services:
+  react:
+    build: ../
+    command: sh -c 'pnpm install && pnpm start --host'
+    environment:
+      APP_TITLE: ${APP_TITLE:-CMS}
+      APP_GRAPHQL_ENDPOINT: ${APP_GRAPHQL_ENDPOINT:-http://localhost:8001/graphql/}
+      APP_GRAPHQL_CODEGEN_ENDPOINT: ${APP_GRAPHQL_CODEGEN_ENDPOINT:-./backend/schema.graphql}
+    volumes:
+      - ../:/code
+    ports:
+      - '3000:3000'
+    depends_on:
+      - web
```

Comment:
Where is this `web` service defined?

Reply:
this is pulled from backend docker-compose